local.go

```diff
@@ -171,14 +173,28 @@ func (l *LocalTest) panicClosed() {
 	}
 }
 
-// CloseAll takes a list of servers that will be closed
+// CloseAll closes all the servers.
 func (l *LocalTest) CloseAll() {
 	log.Lvl3("Stopping all")
 	// If the debug-level is 0, we copy all errors to a buffer that
 	// will be discarded at the end.
 	if log.DebugVisible() == 0 {
 		log.OutputToBuf()
 	}
+
+	if l.T != nil {
```

Comment:
I did it this way to make it possible for protocol tests to "opt-in" to leak checking for the moment. That way we don't have too much stuff crashing all over the place while we experiment with this.